package.json

```diff
@@ -5,32 +5,46 @@
+    "test":
+      "cross-env FORCE_COLOR=1 run-p --aggregate-output test:unit test:integration",
     "test:setup": "ts-node script/test-setup.ts",
     "test:review": "ts-node script/test-review.ts",
-    "postinstall": "cd app && yarn install --force && cd .. && git submodule update --recursive --init && yarn compile:tslint",
+    "postinstall":
+      "cd app && yarn install --force && cd .. && git submodule update --recursive --init",
     "start": "cross-env NODE_ENV=development node script/start",
     "start:prod": "cross-env NODE_ENV=production node script/start",
     "debug": "cross-env NODE_ENV=development node script/debug",
-    "compile:dev": "cross-env NODE_ENV=development parallel-webpack --config app/webpack.development.js",
-    "compile:prod": "cross-env NODE_ENV=production parallel-webpack --config app/webpack.production.js",
-    "build:dev": "yarn compile:dev && cross-env NODE_ENV=development ts-node script/build.ts",
-    "build:prod": "yarn compile:prod && cross-env NODE_ENV=production ts-node script/build.ts",
+    "compile:dev":
+      "cross-env NODE_ENV=development parallel-webpack --config app/webpack.development.js",
+    "compile:prod":
+      "cross-env NODE_ENV=production parallel-webpack --config app/webpack.production.js",
+    "build:dev":
+      "yarn compile:dev && cross-env NODE_ENV=development ts-node script/build.ts",
+    "build:prod":
+      "yarn compile:prod && cross-env NODE_ENV=production ts-node script/build.ts",
     "package": "ts-node -P script script/package.ts",
     "generate-octicons": "ts-node script/generate-octicons.ts",
-    "clean:tslint": "rimraf tslint-rules/*.js",
-    "compile:tslint": "tsc -p tslint-rules",
     "compile:script": "tsc -p script",
-    "lint": "yarn compile:tslint && yarn compile:script && yarn tslint && yarn eslint-check && yarn eslint",
-    "lint:fix": "yarn compile:tslint && yarn tslint:fix && yarn eslint:fix",
-    "tslint": "tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
-    "tslint:fix": "yarn tslint --fix",
+    "lint": "yarn lint:src && yarn lint:prettier",
+    "lint:fix": "cross-env FORCE_COLOR=1 run-p --aggregate-output lint:*:fix",
+    "lint:prettier": "cross-env FORCE_COLOR=1 run-p prettier:*",
+    "lint:prettier:fix": "cross-env FORCE_COLOR=1 run-p 'prettier:* --write'",
+    "prettier:md": "prettier --list-different '**/*.md'",
+    "prettier:scss": "prettier --list-different '**/*.scss'",
+    "prettier:json": "prettier --list-different '**/*.json'",
+    "lint:src": "yarn tslint && yarn eslint-check && yarn eslint",
+    "lint:src:fix": "yarn tslint --fix && yarn eslint --fix",
+    "tslint":
+      "cross-env NODE_OPTIONS='-r ts-node/register' tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
```

Comment:
It's not clear to me why we need `NODE_OPTIONS` here when we're spawning `tslint` - that knows how to parse TS files.

Reply:
`NODE_OPTIONS` is to let us avoid having to transpile our TSLint rules before using them.

Reply:
The whole task takes ~1.5s on my machine, and I couldn't really spot a difference between the three different ways to invoke it. This feels like a bit of yak shaving IMO.

```shellsession
$ time yarn tslint
yarn run v1.3.2
$ cross-env NODE_OPTIONS='-r ts-node/register' tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
✨  Done in 1.51s.
yarn tslint  2.39s user 0.19s system 142% cpu 1.808 total

$ time yarn tslint
yarn run v1.3.2
$ cross-env tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
✨  Done in 1.46s.
yarn tslint  2.43s user 0.17s system 147% cpu 1.768 total

$ time yarn tslint
yarn run v1.3.2
$ tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
✨  Done in 1.35s.
yarn tslint  2.32s user 0.15s system 150% cpu 1.648 total
```